Sources_v3/Database/DTOs/ChannelMemberListQueryDTO.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import CoreData
+
+@objc(ChannelMemberListQueryDTO)
+final class ChannelMemberListQueryDTO: NSManagedObject {
```

Comment:
Was not sure if we want to have the relation to the `channel` here. 

If so, the logic in `ChannelMemberListUpdater` should to updated to fetch the channel from backend first and then perform the query call.

Reply:
I vote for the relationship with the `ChannelDTO`: 
- if one time we decide to have a model for this query, it will require the channel model
- `Filter` is already too complicated and it is for general purpose, not only channel related queries

Reply:
👍

Reply:
fixed in 3429b45af6c0a8f22bff5116ca2b745cb2719351 and 4c3c349d78702726a80fd1eec3f7710080edca1c